problems/loops.js

```diff
@@ -91,7 +158,17 @@ function tillXTimes9ForLoop() {}
  * @returns {number[]} [5, 15, 25, ..., 95]
 
  */
-function endsWithFiveWhileLoop() {}
+function endsWithFiveWhileLoop() {
+  let newArray= []
+  let i = 1
```

Comment:
Another way to write this is to start i at 5 since no number before that will end with 5

Then you can increment i by 5 in a for loop

Just another way to think about it!